web/app/js/components/ErrorModal.jsx

```diff
@@ -124,12 +124,27 @@ export default class ErrorModal extends React.Component {
     });
   }
 
+  renderIconStatus = errors => {
+    let containers = errors.byPodAndContainer;
+    let showInit = _.flatten(_.map(containers, container => {
+      return _.map(container.byContainer, con => {
+        return con[0].reason;
+      });
+    }));
+    if (_.every(showInit, status => status === "PodInitializing")) {
+      return <Badge status="processing" className="controller-init-icon" onClick={this.showModal} />;
```

Comment:
I think if all the pods are initializing, we shouldn't allow the user to click and get a modal - they'll just get the blank error messages template. Maybe just a tooltip saying "Pods Are initializing"? Or if we do want the modal, a message saying "All pods are initializing"?